### ISSUE TITLE
Missing image

### ISSUE THREAD

**Issue by angtheod** [greenpeace/planet4-master-theme#349]
https://github.com/greenpeace/planet4-master-theme/blob/dfae3d984c55cb23d694cc0b6e848da5a888326c/assets/scss/common/_happy-point-block.scss#L3

This image does not exist. So we have a 404 on Home page load which causes some small delay.

@comzeradd @ashish173

**Comment by comzeradd** [comment]
I think this is not used any more with the recent changes to the happy point. @ashish173 is that right?

**Comment by koyan** [comment]
Isn't that the default image that should appear if an editor puts no image there?

**Comment by comzeradd** [comment]
Yes, that could be the case. Let's wait for @ashish173 who is more familiar with the happy-point block. We have that image on the [presentation layer](https://github.com/greenpeace/planet4-presentation-layer/blob/master/images/happy-point-block-bg.jpg) if we still need it.

**Comment by ashish173** [comment]
@koyan is right about the default behaviour of the happy point image. However, we made a change which was to move away from the background image to use the image tag. After this was implemented ideally this code should have been removed.

The default behaviour needs a little more thinking now; setting the default image URL IMO. Then that is picked in the PHP code.

I am proposing we should remove this code in SCSS file and move to the default implementation as I am suggesting above which the backend team can approve or find a better solution for it.

cc: @comzeradd @koyan @angtheod @kirdia
